src/screens/NewsScreen/Header.js

```diff
@@ -126,16 +123,16 @@ const styles = StyleSheet.create({
     backgroundColor: COLORS.gray,
     marginLeft: 6,
     marginRight: 6,
-    marginTop: 1,
+    marginTop: 1
   },
   noPl: {
     padding: 0
   },
   row: {
     flex: 1,
     justifyContent: 'center',
-    marginRight: 8,
+    marginRight: 8
   }
 });
 
-export default React.memo (Header);
+export default React.memo(Header);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There's an unnecessary space between `React.memo` and `(Header)`. While this doesn't cause any syntax error, it's not a common practice and might be confusing.
```diff
- export default React.memo (Header);
+ export default React.memo(Header);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->